assets/scss/_homepage.scss

```diff
@@ -148,4 +148,105 @@ $border-radius: 4px;
     border: 1px solid $colorGrayscale-pebble;
     border-top: none;
   }
+}
+
+.form-give-home-signin {
+  .form-group {
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+
+    > input, button[type=submit] {
+      border-radius: $border-radius;
+    }
+  }
+
+  .form-heading {
+    text-transform: uppercase;
+    pointer-events: none;
+  }
+
+  button[type=submit] {
+    background-color: $colorCru-white;
+    border-color: $colorCru-gold;
+  }
+}
+
+.give-home-slider {
+  // start placeholder for slider
+  height: 397px;
+  width: 100%;
+  background: url("/assets/images/slider image.png") 50% 50%;
```

Comment:
File naming, I know it's not final... avoid spaces, consider hyphen or underscore.